test/com/cloudogu/ces/cesbuildlib/k3dTest.groovy

```diff
@@ -0,0 +1,12 @@
+package com.cloudogu.ces.cesbuildlib
+
+class k3dTest extends GroovyTestCase {
+    void testCreateClusterName() {
+        K3d sut = new K3d("script","workspace", "path")
+        String testClusterName = sut.createClusterName()
+        assertTrue(testClusterName.contains("citest-"))
+        assertTrue(testClusterName.length() <= 32)
```

Comment:
Maybe we should also test that the generated cluster name does not consist only of `citest-`? This way we can see if the token generation failed to deliver a random string, namely delivering the empty string.

This could be done  (for example):
- by length check (testClusterName > len("citest-") or 
- by non-equality check with "citest-"